front/src/app/services/login.service.ts

```diff
@@ -13,9 +13,16 @@ export class LoginService {
 
   constructor(private http: HttpClient) { }
 
-  login(username: string, password: string) {
-    return this.http.post<any>(this.loginUrl, {username, password}).pipe(map(user => user as User));
-    }
+  login(email: string, password: string) {
+    const httpOptions = {
+      headers: new HttpHeaders({
+        'Content-Type':  'application/json',
+        Email: email,
+        Password: password
+      })
+    };
+    return this.http.get<any>(this.loginUrl, httpOptions).pipe(map(user => user as User));
+  }
```

Comment:
Logowanie się używając geta i to z jsonem załączonym w nagłówku nie jest dobrym pomysłem. Powinien być tu użyty post tak, jak było wcześniej.